configure.ac

```diff
@@ -2,7 +2,23 @@
 # Process this file with autoconf to produce a configure script.
 
 AC_PREREQ([2.63])
+
+# IMPORTANT: when editing the version  number, edit both the
+# MARGO_VERSION_ variables and the argument provided to AC_INIT
 AC_INIT([margo], [0.13.1], [],[],[])
+
+MARGO_VERSION_MAJOR=0
```

Comment:
I like having a version.h.  I *think* it might be possible with an m4 macro to reuse one version number definition though, to make sure we don't accidentally get them out of sync.  I'll tinker a little and report back.